necroHeal.lic

```diff
@@ -0,0 +1,171 @@
+=begin

```

Comment:
Our convention has been to put documentation on elanthipedia and have a script header like this:

```
=begin
  Documentation: https://elanthipedia.play.net/Lich_script_development#common-healing
=end
```